fm.reaper.Reaper.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Reaper
+Exec=reaper
+# Icon is validated at build time, but it doesn't extist yet because apply_extra
+# hasn't run yet.
+# Icon=/app/extra/REAPER/Resources/main.png
```

Comment:
you need to actually add an icon here too.

Reply:
@hfiguiere For some reason, I can't place the icon for apparently some lack of permission (?).
`install: cannot create directory '/app/share/icons': Read-only file system`

Reply:
Change this apparently fixed it:
```
-           - install -Dm644 /app/extra/REAPER/Resources/main.png /app/share/icons/hicolor/${size}x${size}/apps/${FLATPAK_ID}.png
+           - install -Dm644 /app/extra/REAPER/Resources/main.png $FLATPAK_APP_DIR/share/icons/hicolor/${size}x${size}/apps/${FLATPAK_ID}.png

```

Reply:
But still can't find the icon. :/